### DISCUSSION TITLE
Should Schlage lock show up in homekit via this plugin?

### DISCUSSION THREAD

**Discussion by vronp** [dgreif/ring#1359] [Q&A]
Hi,

I have a z-wave Schlage lock that appears in the Ring app.  Should I be able to see this lock in HomeKit as a result of using this plug-in?  Thanks very much.

**Comment by tsightler** [comment]
If it's a Z-wave locked paired with Ring Alarm, yes, otherwise, no.

**Reply by vronp** [reply]
By "paired" I assume you mean "linked" to use the Ring terminology?  If so, do you have any suggestions for investigating why the lock isn't showing up in homekit?  Do I need to add a device type to this plugin perhaps?

**Reply by tsightler** [reply]
I've never really seen Ring use the term linked, unless it was with linking an external account.  Perhaps you can provide a link to the documentation for the process you used.  Regardless, it's pretty simple, if the device is connected directly to your Ring Alarm Base station via Z-wave, then it should be supported.

Best suggestion for troubleshooting is the restart the addon and look at the logs and see if the device is listed and also if the device is listed in Homebridge Accessories UI.

**Reply by vronp** [reply]
Ok, I understand now.  Linked means linking an external account, in my case, the Schlage and MyQ app.  Neither the door lock or garage door show up on HomeKit.  I'm assuming there is no Z-wave connection but I will try to verify.  Thanks very much.

**Reply by tsightler** [reply]
What model Schlage lock do you have?  MyQ is not connected to any Ring device and it's just an app link.

**Reply by vronp** [reply]
BE489WB

**Reply by tsightler** [reply]
The BE489WB is a Schlage Encode lock, these are Wifi locks, not Z-wave, thus are not supported by this plugin as they do not connect via a Ring device.
